### PR TITLE
Feat : added notes component and grid layout

### PR DESCRIPTION
**Pull request by yogesh-**
image

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -46,9 +49,77 @@
   border-width: 0 0 0.2rem;
   border-color: whitesmoke;
   background-color: transparent;
+  color: whitesmoke;
+  font-size: 2rem;
 }
 
 #cbox:checked + label {
-  color: whitesmoke;
+  color: rgb(240, 119, 119);
```
Comment (SRUJANA-PENUGONDA13) [inline src/App.css]: You can hash code as well

File: src/components/notes.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import "../App.css";
+import { useState } from "react";
+
+const Note = () => {
+  const [show, setShow] = useState(false);
```
Comment (SRUJANA-PENUGONDA13) [inline src/components/notes.js]: Try to give this variable name more descriptive. I didn't strike immediately. I think you can use displayStatus or todoDisplayStatus

**Review by SRUJANA-PENUGONDA13 — COMMENTED** [review]
Everything looks good. Added few suggestions once go through it.

File: src/components/notes.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  const noteAdd = (e) => {
+    setNote("");
+    setnotesArray([...notesArray, note]);
+  };
+
+  return (
+    <>
+      {" "}
+      <button onClick={(e) => clickHandler(e)}>ToDo</button>
+      {show ? (
```
Comment (VishakAmin) [inline src/components/notes.js]: If you need to satisfy only one condition use && instead of ternary oprerator.

File: src/components/notes.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        <div className="notes-bg">
+          <p>My Todo's</p>
+          {notesArray.map((item) => (
+            <li>
```
Comment (VishakAmin) [inline src/components/notes.js]: **_key_** is missing

File: src/components/notes.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  const [note, setNote] = useState();
+  const [notesArray, setnotesArray] = useState([]);
+  const clickHandler = (e) => {
+    if (show !== true) {
+      setShow(true);
+    } else {
+      setShow(false);
+    }
+  };
+
```
Comment (VishakAmin) [inline src/components/notes.js:15]: if you want to switch the boolean based on the previous it can be done like this 
`setShow(show => !show)`

**Review by VishakAmin — COMMENTED** [review]
Left some comments please have alook

File: src/components/notes.js (hunk 5 above)
Comment (yogesh-) [reply src/components/notes.js:15]: okay

File: src/App.css (hunk 1 above)
Comment (yogesh-) [reply src/App.css]: ok

File: src/components/notes.js (hunk 2 above)
Comment (yogesh-) [reply src/components/notes.js]: done

File: src/components/notes.js (hunk 3 above)
Comment (yogesh-) [reply src/components/notes.js]: ok

File: src/components/notes.js (hunk 4 above)
Comment (yogesh-) [reply src/components/notes.js]: ok

**Pull request merged**